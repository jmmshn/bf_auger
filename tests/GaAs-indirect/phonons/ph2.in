GaAs phonons
&inputph
  tr2_ph   =  1.0d-14
  prefix   = 'GaAs'
  outdir   = './'
  fildyn   = 'GaAs.dyn'
  epsil    = .false.
  fildvscf = 'dvscf'
  trans    = .false.
  electron_phonon = 'yambo'
  recover  = .false.
  verbosity = 'high'
 /
QX QY QZ
