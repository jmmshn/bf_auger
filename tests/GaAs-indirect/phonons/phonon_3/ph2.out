
     Program PHONON v.6.2 (svn rev. 13949:13950) starts on 24Dec2018 at  9:42:43 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on     272 processor cores
     Number of MPI processes:                68
     Threads/MPI process:                     4

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      68

     Reading data from directory:
     ./GaAs.save

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = LDA ( 1  1  0  0 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file Ga.pz-dn-rrkjus_psl.0.2.UPF: wavefunction(s)  4S 3D renormalized
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          11      11      2                  208      208      23
     Max          12      12      3                  212      212      29
     Sum         769     769    199                14211    14211    1807
 

     Check: negative/imaginary core charge=   -0.000007    0.000000
  autoval = -.8583E+01
     Irreps are as follows:

                     mode #   1
                     (  -0.57841  -0.00000   ) 
                     (   0.57841  -0.00000   ) 
                     (  -0.57511  -0.01209   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
  autoval = -.5045E+01
     Irreps are as follows:

                     mode #   2
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.70711   0.00000   ) 
                     (  -0.70711  -0.00000   ) 
                     (  -0.00000  -0.00000   ) 
  autoval = 0.3314E+00
     Irreps are as follows:

                     mode #   3
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.69415   0.00000   ) 
                     (   0.69415  -0.00000   ) 
                     (   0.15926   0.10466   ) 
  autoval = 0.1082E+01
     Irreps are as follows:

                     mode #   4
                     (  -0.40675  -0.00000   ) 
                     (   0.40675  -0.00000   ) 
                     (   0.81781   0.01720   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
  autoval = 0.5281E+01
     Irreps are as follows:

                     mode #   5
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.13475  -0.00000   ) 
                     (   0.13475   0.00000   ) 
                     (  -0.82039  -0.53912   ) 
  autoval = 0.1554E+02
     Irreps are as follows:

                     mode #   6
                     (  -0.70711   0.00000   ) 
                     (  -0.70711   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 

     Calculation of q =    0.1666667  -0.1666667   0.5000000
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          11      11      3                  208      208      29
     Max          12      12      4                  212      212      32
     Sum         769     769    211                14211    14211    2085
 


     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.6031  a.u.
     unit-cell volume          =     298.0132 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     number of electrons       =        18.00
     number of Kohn-Sham states=           20
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     Exchange-correlation      = LDA ( 1  1  0  0 0 0)

     celldm(1)=  10.603075  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Ga read from file:
     ../PP_DIR/Ga.pz-dn-rrkjus_psl.0.2.UPF
     MD5 check sum: 337958ae198bc3768054a43796a587a1
     Pseudo is Ultrasoft + core correction, Zval = 13.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Using radial grid of 1205 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for As read from file:
     ../PP_DIR/As.pz-n-rrkjus_psl.0.2.UPF
     MD5 check sum: f80104272e9b817acfe428865afbdb7e
     Pseudo is Ultrasoft + core correction, Zval =  5.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Using radial grid of 1209 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Ga            13.00    69.72300     Ga( 1.00)
        As             5.00    74.92200     As( 1.00)

     24 Sym. Ops. (no inversion) found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000 -0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  2     inv. 180 deg rotation - cart. axis [1,1,0]   

 cryst.   s( 2) = (     0          1          0      )
                  (     1          0          0      )
                  (     0          0          1      )

 cart.    s( 2) = ( -0.0000000 -1.0000000 -0.0000000 )
                  ( -1.0000000 -0.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000  1.0000000 )


      isym =  3     180 deg rotation - cart. axis [0,1,0]        

 cryst.   s( 3) = (    -1          0          0      )
                  (    -1          0          1      )
                  (    -1          1          0      )

 cart.    s( 3) = ( -1.0000000  0.0000000  0.0000000 )
                  ( -0.0000000  1.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000 -1.0000000 )


      isym =  4     180 deg rotation - cart. axis [1,0,0]        

 cryst.   s( 4) = (     0         -1          1      )
                  (     0         -1          0      )
                  (     1         -1          0      )

 cart.    s( 4) = (  1.0000000 -0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000 -1.0000000 )


      isym =  5     120 deg rotation - cart. axis [-1,-1,-1]     

 cryst.   s( 5) = (    -1          0          1      )
                  (    -1          1          0      )
                  (    -1          0          0      )

 cart.    s( 5) = ( -0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym =  6     120 deg rotation - cart. axis [-1,1,1]       

 cryst.   s( 6) = (     0          1          0      )
                  (     0          0          1      )
                  (     1          0          0      )

 cart.    s( 6) = (  0.0000000 -1.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000  1.0000000 )
                  ( -1.0000000 -0.0000000  0.0000000 )


      isym =  7     120 deg rotation - cart. axis [1,1,-1]       

 cryst.   s( 7) = (     1          0         -1      )
                  (     0          0         -1      )
                  (     0          1         -1      )

 cart.    s( 7) = (  0.0000000  1.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000 -1.0000000 )
                  ( -1.0000000  0.0000000 -0.0000000 )


      isym =  8     120 deg rotation - cart. axis [1,-1,1]       

 cryst.   s( 8) = (     0         -1          0      )
                  (     1         -1          0      )
                  (     0         -1          1      )

 cart.    s( 8) = ( -0.0000000 -1.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000 -1.0000000 )
                  (  1.0000000 -0.0000000 -0.0000000 )


      isym =  9     120 deg rotation - cart. axis [1,1,1]        

 cryst.   s( 9) = (     0          0         -1      )
                  (     0          1         -1      )
                  (     1          0         -1      )

 cart.    s( 9) = (  0.0000000 -0.0000000  1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 10     120 deg rotation - cart. axis [-1,1,-1]      

 cryst.   s(10) = (    -1          1          0      )
                  (    -1          0          0      )
                  (    -1          0          1      )

 cart.    s(10) = ( -0.0000000  0.0000000  1.0000000 )
                  ( -1.0000000 -0.0000000  0.0000000 )
                  ( -0.0000000 -1.0000000  0.0000000 )


      isym = 11     120 deg rotation - cart. axis [1,-1,-1]      

 cryst.   s(11) = (     0          0          1      )
                  (     1          0          0      )
                  (     0          1          0      )

 cart.    s(11) = ( -0.0000000 -0.0000000 -1.0000000 )
                  ( -1.0000000  0.0000000 -0.0000000 )
                  ( -0.0000000  1.0000000 -0.0000000 )


      isym = 12     120 deg rotation - cart. axis [-1,-1,1]      

 cryst.   s(12) = (     1         -1          0      )
                  (     0         -1          1      )
                  (     0         -1          0      )

 cart.    s(12) = (  0.0000000  0.0000000 -1.0000000 )
                  (  1.0000000 -0.0000000 -0.0000000 )
                  (  0.0000000 -1.0000000 -0.0000000 )


      isym = 13     180 deg rotation - cart. axis [0,0,1]        

 cryst.   s(13) = (     0          1         -1      )
                  (     1          0         -1      )
                  (     0          0         -1      )

 cart.    s(13) = ( -1.0000000 -0.0000000 -0.0000000 )
                  ( -0.0000000 -1.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000  1.0000000 )


      isym = 14     inv. 180 deg rotation - cart. axis [1,-1,0]  

 cryst.   s(14) = (     1          0         -1      )
                  (     0          1         -1      )
                  (     0          0         -1      )

 cart.    s(14) = (  0.0000000  1.0000000 -0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym = 15     inv.  90 deg rotation - cart. axis [0,0,-1]  

 cryst.   s(15) = (     0         -1          0      )
                  (     0         -1          1      )
                  (     1         -1          0      )

 cart.    s(15) = (  0.0000000 -1.0000000  0.0000000 )
                  (  1.0000000 -0.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000 -1.0000000 )


      isym = 16     inv.  90 deg rotation - cart. axis [0,0,1]   

 cryst.   s(16) = (    -1          0          1      )
                  (    -1          0          0      )
                  (    -1          1          0      )

 cart.    s(16) = ( -0.0000000  1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000 -1.0000000 )


      isym = 17     inv. 180 deg rotation - cart. axis [1,0,1]   

 cryst.   s(17) = (     1          0          0      )
                  (     0          0          1      )
                  (     0          1          0      )

 cart.    s(17) = (  0.0000000  0.0000000 -1.0000000 )
                  ( -0.0000000  1.0000000 -0.0000000 )
                  ( -1.0000000  0.0000000 -0.0000000 )


      isym = 18     inv. 180 deg rotation - cart. axis [-1,0,1]  

 cryst.   s(18) = (    -1          0          0      )
                  (    -1          1          0      )
                  (    -1          0          1      )

 cart.    s(18) = ( -0.0000000  0.0000000  1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym = 19     inv.  90 deg rotation - cart. axis [0,1,0]   

 cryst.   s(19) = (     0         -1          1      )
                  (     1         -1          0      )
                  (     0         -1          0      )

 cart.    s(19) = ( -0.0000000 -0.0000000 -1.0000000 )
                  (  0.0000000 -1.0000000 -0.0000000 )
                  (  1.0000000 -0.0000000 -0.0000000 )


      isym = 20     inv.  90 deg rotation - cart. axis [0,-1,0]  

 cryst.   s(20) = (     0          1         -1      )
                  (     0          0         -1      )
                  (     1          0         -1      )

 cart.    s(20) = (  0.0000000 -0.0000000  1.0000000 )
                  ( -0.0000000 -1.0000000  0.0000000 )
                  ( -1.0000000 -0.0000000  0.0000000 )


      isym = 21     inv. 180 deg rotation - cart. axis [0,1,1]   

 cryst.   s(21) = (     1         -1          0      )
                  (     0         -1          0      )
                  (     0         -1          1      )

 cart.    s(21) = (  1.0000000  0.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000 -1.0000000 )
                  (  0.0000000 -1.0000000 -0.0000000 )


      isym = 22     inv. 180 deg rotation - cart. axis [0,1,-1]  

 cryst.   s(22) = (     0          0          1      )
                  (     0          1          0      )
                  (     1          0          0      )

 cart.    s(22) = (  1.0000000 -0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 23     inv.  90 deg rotation - cart. axis [-1,0,0]  

 cryst.   s(23) = (     0          0         -1      )
                  (     1          0         -1      )
                  (     0          1         -1      )

 cart.    s(23) = ( -1.0000000 -0.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000 -1.0000000 )
                  ( -0.0000000  1.0000000 -0.0000000 )


      isym = 24     inv.  90 deg rotation - cart. axis [1,0,0]   

 cryst.   s(24) = (    -1          1          0      )
                  (    -1          0          1      )
                  (    -1          0          0      )

 cart.    s(24) = ( -1.0000000  0.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000  1.0000000 )
                  ( -0.0000000 -1.0000000  0.0000000 )


     point group T_d (-43m) 
     there are  5 classes
     the character table:

       E     8C3   3C2   6S4   6s_d 
A_1    1.00  1.00  1.00  1.00  1.00
A_2    1.00  1.00  1.00 -1.00 -1.00
E      2.00 -1.00  2.00  0.00  0.00
T_1    3.00  0.00 -1.00  1.00 -1.00
T_2    3.00  0.00 -1.00 -1.00  1.00

     the symmetry operations in each class and the name of the first element:

     E        1
          identity                                               
     8C3      5   12    8    6    7    9   10   11
          120 deg rotation - cart. axis [-1,-1,-1]               
     3C2      3    4   13
          180 deg rotation - cart. axis [0,1,0]                  
     6S4     15   16   20   19   23   24
          inv.  90 deg rotation - cart. axis [0,0,-1]            
     6s_d     2   14   17   18   21   22
          inv. 180 deg rotation - cart. axis [1,1,0]             

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Ga  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           As  tau(   2) = (  -0.7500000   0.7500000   0.7500000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Ga  tau(   1) = (  0.0000000  0.0000000  0.0000000  )
         2           As  tau(   2) = (  0.7500000  0.7500000  0.7500000  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000
        k(    2) = (   0.1666667  -0.1666667   0.5000000), wk =   0.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000
        k(    2) = (   0.1666667   0.1666667  -0.1666667), wk =   0.0000000

     Dense  grid:    14211 G-vectors     FFT dimensions: (  36,  36,  36)

     Dynamical RAM for                 wfc:       0.01 MB

     Dynamical RAM for     wfc (w. buffer):       0.01 MB

     Dynamical RAM for           str. fact:       0.01 MB

     Dynamical RAM for           local pot:       0.00 MB

     Dynamical RAM for          nlocal pot:       0.01 MB

     Dynamical RAM for                qrad:       3.92 MB

     Dynamical RAM for          rho,v,vnew:       0.04 MB

     Dynamical RAM for           G-vectors:       0.01 MB

     Dynamical RAM for          h,s,v(r/c):       0.29 MB

     Dynamical RAM for          <psi|beta>:       0.01 MB

     Dynamical RAM for                 psi:       0.03 MB

     Dynamical RAM for                hpsi:       0.03 MB

     Dynamical RAM for                spsi:       0.03 MB

     Dynamical RAM for      wfcinit/wfcrot:       0.03 MB

     Dynamical RAM for           addusdens:       0.60 MB

     Estimated static dynamical RAM per process >       4.06 MB

     Estimated max dynamical RAM per process >       4.66 MB

     Estimated total dynamical RAM >     316.94 MB
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization
     Message from routine sym_rho_init:
     some processors have no G-vectors for symmetrization

     Check: negative/imaginary core charge=   -0.000007    0.000000

     The potential is recalculated from file :
     ./_ph0/GaAs.save/charge-density.dat

     Starting wfc are   13 atomic +    7 random wfc

     Band Structure Calculation
     Davidson diagonalization with overlap

     Computing kpt #:     1
     total cpu time spent up to now is        0.9 secs

     Computing kpt #:     2
     total cpu time spent up to now is        1.5 secs

     ethr =  5.56E-11,  avg # of iterations = 14.5

     total cpu time spent up to now is        1.5 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000 (  1807 PWs)   bands (ev):

    -8.2061  -8.2061  -8.2061  -8.1239  -8.1239  -6.1701   6.7912   6.7912
     6.7912   7.2369  10.4290  10.4290  10.4290  14.2452  17.0280  17.0280
    18.3527  18.3527  18.3527  21.1452

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000

          k = 0.1667-0.1667 0.5000 (  1789 PWs)   bands (ev):

    -8.2734  -8.1844  -8.1817  -8.1325  -8.1171  -5.0367   1.9120   4.8318
     5.1466   9.1868  10.5085  12.3494  12.8472  15.6139  17.5391  17.8374
    19.8487  22.1875  23.5122  23.7664

     occupation numbers 
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000

     highest occupied, lowest unoccupied level (ev):     6.7912    7.2369

     Writing output data file GaAs.save

                                                                                

     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.6031  a.u.
     unit-cell volume          =     298.0132 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     kinetic-energy cut-off    =      50.0000  Ry
     charge density cut-off    =     200.0000  Ry
     convergence threshold     =      1.0E-14
     beta                      =       0.7000
     number of iterations used =            4
     Exchange-correlation      = LDA ( 1  1  0  0 0 0)


     celldm(1)=   10.60307  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of alat)
               a(1) = ( -0.5000  0.0000  0.5000 )  
               a(2) = (  0.0000  0.5000  0.5000 )  
               a(3) = ( -0.5000  0.5000  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.0000 -1.0000  1.0000 )  
               b(2) = (  1.0000  1.0000  1.0000 )  
               b(3) = ( -1.0000  1.0000 -1.0000 )  


     Atoms inside the unit cell: 

     Cartesian axes

     site n.  atom      mass           positions (alat units)
        1     Ga  69.7230   tau(    1) = (    0.00000    0.00000    0.00000  )
        2     As  74.9220   tau(    2) = (   -0.75000    0.75000    0.75000  )

     Computing dynamical matrix for 
                    q = (   0.1666667  -0.1666667   0.5000000 )
 
      2 Sym.Ops. (no q -> -q+G )

                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000 -0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  2     inv. 180 deg rotation - cart. axis [1,1,0]   

 cryst.   s( 2) = (     0          1          0      )
                  (     1          0          0      )
                  (     0          0          1      )

 cart.    s( 2) = ( -0.0000000 -1.0000000 -0.0000000 )
                  ( -1.0000000 -0.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000  1.0000000 )


     G cutoff =  569.5527  (    212 G-vectors)     FFT grid: ( 36, 36, 36)
     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000
        k(    2) = (   0.1666667  -0.1666667   0.5000000), wk =   0.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000
        k(    2) = (   0.1666667   0.1666667  -0.1666667), wk =   0.0000000

     PseudoPot. # 1 for Ga read from file:
     ../PP_DIR/Ga.pz-dn-rrkjus_psl.0.2.UPF
     MD5 check sum: 337958ae198bc3768054a43796a587a1
     Pseudo is Ultrasoft + core correction, Zval = 13.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Using radial grid of 1205 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for As read from file:
     ../PP_DIR/As.pz-n-rrkjus_psl.0.2.UPF
     MD5 check sum: f80104272e9b817acfe428865afbdb7e
     Pseudo is Ultrasoft + core correction, Zval =  5.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Using radial grid of 1209 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     point group C_s (m)    
     there are  2 classes
     the character table:

       E     s    
A'     1.00  1.00
A''    1.00 -1.00

     the symmetry operations in each class and the name of the first element:

     E        1
          identity                                               
     s        2
          inv. 180 deg rotation - cart. axis [1,1,0]             


     Atomic displacements:
     There are   6 irreducible representations

     Representation     1      1 modes -A'  To be done
     Irreps are as follows:

                     mode #   1
                     (  -0.57841  -0.00000   ) 
                     (   0.57841  -0.00000   ) 
                     (  -0.57511  -0.01209   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 

     Representation     2      1 modes -A'  To be done
     Irreps are as follows:

                     mode #   2
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.69415   0.00000   ) 
                     (   0.69415  -0.00000   ) 
                     (   0.15926   0.10466   ) 

     Representation     3      1 modes -A'  To be done
     Irreps are as follows:

                     mode #   3
                     (  -0.40675  -0.00000   ) 
                     (   0.40675  -0.00000   ) 
                     (   0.81781   0.01720   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 

     Representation     4      1 modes -A'  To be done
     Irreps are as follows:

                     mode #   4
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.13475  -0.00000   ) 
                     (   0.13475   0.00000   ) 
                     (  -0.82039  -0.53912   ) 

     Representation     5      1 modes -A''  To be done
     Irreps are as follows:

                     mode #   5
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (  -0.70711   0.00000   ) 
                     (  -0.70711  -0.00000   ) 
                     (  -0.00000  -0.00000   ) 

     Representation     6      1 modes -A''  To be done
     Irreps are as follows:

                     mode #   6
                     (  -0.70711   0.00000   ) 
                     (  -0.70711   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 
                     (   0.00000   0.00000   ) 


     PHONON       :    11.49s CPU         6.43s WALL

     Reading dVscf from file dvscf1_
     Reading dynamics matrix from file GaAs.dyn1
     electron-phonon interaction  ...

 
     PHONON       :  0m15.50s CPU     1m 9.05s WALL

     INITIALIZATION: 
     phq_setup    :      0.02s CPU      0.03s WALL (       1 calls)
     phq_init     :      0.65s CPU      0.22s WALL (       1 calls)
 
     phq_init     :      0.65s CPU      0.22s WALL (       1 calls)
     set_drhoc    :      0.03s CPU      0.03s WALL (       1 calls)
     init_vloc    :      0.01s CPU      0.01s WALL (       2 calls)
     init_us_1    :      1.42s CPU      0.47s WALL (       2 calls)
     newd         :      0.07s CPU      0.02s WALL (       2 calls)
     dvanqq       :      0.27s CPU      0.07s WALL (       2 calls)
     drho         :      0.46s CPU      0.14s WALL (       1 calls)
 
 
 
 
     dvqpsi_us    :      0.68s CPU      0.19s WALL (       6 calls)
     addusddens   :      0.18s CPU      0.05s WALL (       6 calls)
     newdq        :      0.23s CPU      0.06s WALL (       6 calls)
     adddvscf     :      0.01s CPU      0.00s WALL (       6 calls)
 
     dvqpsi_us    :      0.68s CPU      0.19s WALL (       6 calls)
     dvqpsi_us_on :      0.17s CPU      0.05s WALL (       6 calls)
 
 
     h_psi        :      1.89s CPU      0.56s WALL (      33 calls)
 
     h_psi        :      1.89s CPU      0.56s WALL (      33 calls)
     add_vuspsi   :      0.04s CPU      0.01s WALL (      33 calls)
 
     addusdbec    :      0.02s CPU      0.00s WALL (       6 calls)
 
 
      General routines
     calbec       :      0.14s CPU      0.06s WALL (      47 calls)
     fft          :      0.23s CPU      0.16s WALL (      24 calls)
     ffts         :      0.04s CPU      0.01s WALL (      12 calls)
     fftw         :      2.80s CPU      0.74s WALL (    1406 calls)
     davcio       :      0.48s CPU      0.19s WALL (      19 calls)
 
 
     PHONON       :  0m15.50s CPU     1m 9.05s WALL

 
   This run was terminated on:   9:43:52  24Dec2018            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
